clients/go/internal/containersuite/containerSuite.go

```diff
@@ -175,6 +175,7 @@ func (s *ContainerSuite) SetupSuite() {
 			Image:        s.ContainerImage,
 			ExposedPorts: []string{"26500"},
 			WaitingFor:   zeebeWaitStrategy{waitTime: s.WaitTime},
+			Env:          map[string]string{"ZEEBE_HOST": "0.0.0.0"},
```

Comment:
I know this is supported, but using `ZEEBE_HOST` is deprecated, and we should use `ZEEBE_BROKER_NETWORK_HOST` and `ZEEBE_BROKER_NETWORK_ADVERTISEDHOST` (since this is what shows up in the topology).